updater/lib/dependabot/service.rb

```diff
@@ -19,7 +19,7 @@ class Service
     extend T::Sig
     extend Forwardable
 
-    sig { returns(T::Array[T::Array[String]]) }
+    sig { returns(T::Array[T.untyped]) }
```

Comment:
Is this a client specific response? Too difficult to type without a common interface?

Reply:
`pull_requests` is something used only in this file and it's tricky to type, so I think we can circle back eventually.